docs(tutorial): waypoint plugin for scaleway (#1827)
scaleway/docs-content: tutorials/waypoint-plugin-scaleway/index.mdx

content:
  h1: Configuring Scaleway plugins for Waypoint
  paragraph: In this tutorial you will learn how to install Scaleway plugins for Waypoint
tags: waypoint plugin container
categories:
  - serverless
  - containers
  - kubernetes
dates:
  validation: 2023-06-15
  posted: 2023-06-15
---

Waypoint is an open-source tool developed by HashiCorp that focuses on simplifying the deployment and release workflows for applications.

The main goal of Waypoint is to abstract away the complexities of different deployment targets and provide a consistent interface for developers and operators.
It allows developers to deploy, manage, and observe their applications using a simple and declarative configuration file.
The tool works with Nomad, Kubernetes, ECS and many other platforms.

## Waypoint plugins for Scaleway

We provide plugins for Waypoint, that add support for Scaleway.
Currently, the only plugin available is `container`.

## Plugin installation

### From releases

1. [Download the zip](https://github.com/scaleway/waypoint-plugin-scaleway/releases) of the latest version for your architecture.

> 2. Unzip the plugin by running the following command: `unzip waypoint-plugin-scaleway-container_*.zip -d ~/.config/.waypoint/plugins/`

Reviewer: I think you should invert the actions : "run .... to unzip ..."

Author: ```suggestion
2. Run the following command to unzip the plugin: `unzip waypoint-plugin-scaleway-container_*.zip -d ~/.config/.waypoint/plugins/`
```

    <Message type="note">

>       On macOS, you will have to execute the following command to ignore Apple's developer authenticity verification

Reviewer: ```suggestion
      If you are using macOS, you will have to execute the following command to ignore Apple's developer authenticity verification
```
